case_workflows/workflows/ciaa_caseworker/etc/scripts/download_ciaa_press_releases.py

```
@@ -0,0 +1,45 @@
+OUTPUT_DIR = ".agents/caseworker/data"
+OUTPUT_FILE = os.path.join(OUTPUT_DIR, "ciaa-press-releases.csv")
+
+FIELDS = ["press_id", "publication_date", "title", "source_url"]
+
+
+def fetch_all_manuscripts(index_url):
+    manuscripts = []
+    url = index_url
+    while url:
+        print(f"Fetching: {url}")
+        resp = requests.get(url)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)